### PR TITLE
Update eslint-config-prettier to 8.1.0

### PR DESCRIPTION
**Pull request by bostrom**
Makes now included prettier/react extension obsolete.

### CODE REVIEW

**Comment by bostrom** [comment]
@petetnt not sure how to test this, as it doesn't seem to play well with `yarn link`...

**Comment by petetnt** [comment]
I usually publish alpha versions to npm and test those out

File: packages/eslint-config-motley/package.json [hunk 1]
```diff
@@ -24,7 +24,7 @@
     "eslint": "^7.7.0",
     "eslint-config-airbnb": "18.2.0",
     "eslint-config-airbnb-base": "14.2.0",
-    "eslint-config-prettier": "^6.11.0",
+    "eslint-config-prettier": "^8.1.0",
```
Comment (petetnt) [inline packages/eslint-config-motley/package.json:27]: https://github.com/motleyagency/eslint-config-motley/blob/master/packages/eslint-config-motley-typescript/.eslintrc.js#L5

Does this need to be removed too? Probably as it's already in the main config.

Comment (bostrom) [reply packages/eslint-config-motley/package.json:27]: Right, good catch.

**Comment by petetnt** [comment]
Also remember to update https://github.com/motleyagency/eslint-config-motley/blob/master/VERSIONS.md :)

**Review by petetnt — CHANGES_REQUESTED** [review]
https://github.com/motleyagency/eslint-config-motley/blob/master/packages/eslint-config-motley-typescript/package.json#L29 this needs to be bumped too

**Review by petetnt — APPROVED** [review]
Tested with the changes, LGTM, seems to work

**Comment by petetnt** [comment]
You can either merge this and then bump the version to stable or just bump the version here and rebase & merge

**Comment by bostrom** [comment]
How about squash and merge so we don't get all those bs intermediate commits? Then bump to stable in master?

**Comment by petetnt** [comment]
Sure

**Pull request merged**